Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -187,6 +187,16 @@ public void AddColumn( DataGridColumn<TItem> column )
             }
         }
 
+        /// <summary>
+        /// Removes an existing link of a child column with this datagrid.
+        /// <para>Returns:
+        ///     true if item is successfully removed; otherwise, false. 
+        /// </para>
+        /// </summary>
+        /// <param name="column">Column to link with this datagrid.</param>
+        public bool RemoveColumn( DataGridColumn<TItem> column )

```

Comment:
Should we add the `public` method? While not breaking-change it still changes the API.

Reply:
I forgot. Let's make it internal. And introduce it public on 9.5

Reply:
Done!